client/src/components/collectionPanel/CollectionSettingsTab/CollectionSettingsTab.tsx

```diff
@@ -0,0 +1,44 @@
+import { IconButton } from '@chakra-ui/react';
+import { useState } from 'react';
+import { VscSave } from 'react-icons/vsc';
+
+import GroupsInput from '../../groupsInput';
+
+type CollectionSettingsTabProps = {
+  groups: string[];
+  setGroups: (groups: string[]) => void;
+};
+
+type SettingsTabState = {
+  rawGroups: string[];
+};
+
+export default function CollectionSettingsTab({
+  groups,
+  setGroups,
+}: CollectionSettingsTabProps) {
+  const [state, setState] = useState<SettingsTabState>({
+    rawGroups: [...groups],
+  });
+
+  const handleSaveCollectionSettingsClicked = () => {
+    setGroups(state.rawGroups);
+  };
+
+  return (
+    <div>
+      <GroupsInput
+        groups={state.rawGroups}
+        setGroups={(rawGroups: string[]) => setState({ ...state, rawGroups })}
+        isRounded
+      />
+      <IconButton
```

Comment:
I think this intermediate save button will be confusing for users, as they will think that after clicking it, the groups are persisted, when in reality it's only stored locally in the CurrentCollection. I think we can get rid of the state here and simply pass `setGroups` to the GroupsInput directly. This should also resolve the bug of having inconsistent state in the collections.

Saving the current collection to the database will then be done via the collection-save button.

Reply:
Yea, that makes total sense. I forgot that we have already a save mechanism for collections. 
Will change the MR accordingly.